docs/meta/README.md

```diff
@@ -0,0 +1,6 @@
+The meta folder contains docs not about how to **use** Beaker, but how to
+deal with Beaker in other ways, such as how to **change** it.
+
+**Table of Contents**
+
+- [Ticket Process](docs/meta/ticket_process.md)
```

Comment:
thanks for catching that @tvpartytonight! I'd meant to do a quick validate after pushing and before submitting the PR, but spaced on it. Links should be good now